chainer/functions/noise/dropout.py

```diff
@@ -105,9 +106,28 @@ def dropout(x, ratio=.5, **kwargs):
             Input variable. A :math:`(s_1, s_2, ..., s_N)` -shaped float array.
         ratio (float):
             Dropout ratio. The ``ratio`` must be ``0.0 <= ratio < 1.0``.
+        mask (ndarray or None):
+            The mask to be used for dropout.
+            The shape and dtype must be the same as ``x`` and should be on the
+            same device.
+            If ``mask`` is not specified or set to ``None``, a mask will be
+            generated randomly according to the given ``ratio``.
+            If ``mask`` is specified, ``ratio`` will be ignored.
+            Note that iDeep will not be used for this function if mask is
+            specified, as iDeep does not support it.
+        return_mask (bool):
+            If ``True``, the mask used for dropout is returned altogether with
```

Comment:
A nitpick but could you change "altogether" to "together"?

Reply:
Fixed.